Order.Management/ToyBlocks/Shape.cs

```diff
@@ -1,17 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Order.Management
+﻿namespace Order.Management.ToyBlocks
 {
-    abstract class Shape
+    // TODO: rename this to Block
+    public abstract class Shape
     {
         public string Name { get; set; }
         public int Price { get; set; }
         public int AdditionalCharge { get; set; }
```

Comment:
The current data structure only allows for one `AdditionalCharge` per shape. 

Possible solution is for each attribute of the shape to have some `AdditionalCharge` (i.e. the colour and shape type)